### ISSUE TITLE
Tensorflow Object detection training finishes at Step 0, never move to Step 1. 

### ISSUE THREAD

**Issue by SakuSakuLab** [tensorflow/models#7526]
### System information
- **What is the top-level directory of the model you are using**:/object_detection
- **Have I written custom code (as opposed to using a stock example script provided in TensorFlow)**: No
- **OS Platform and Distribution (e.g., Linux Ubuntu 16.04)**: Win10
- **TensorFlow installed from (source or binary)**: binary
- **TensorFlow version (use command below)**: tensorflow-gpu 1.12.0
- **Bazel version (if compiling from source)**: N/A
- **CUDA/cuDNN version**:  V9.0.176
- **GPU model and memory**: GeForce RTX 2080 (x2)
- **Exact command to reproduce**: python train.py --logtostderr --train_dir=training/ --pipeline_config_path=training/ssd_inception_v2_coco.config

### Describe the problem
Training finises at Step 0, never move to Step 1. I just followed [tensorflow object detection api tutorial](https://tensorflow-object-detection-api-tutorial.readthedocs.io/en/latest/training.html). In Training Custom Object Detector section, I've got a problem. I used the ssd_inception_v2 algorithm. 

When I execute above command line, I got a message below.

### Source code / logs
........
name: GeForce RTX 2080 major: 7 minor: 5 memoryClockRate(GHz): 1.83
pciBusID: 0000:17:00.0
totalMemory: 8.00GiB freeMemory: 6.55GiB
2019-09-03 16:13:53.388657: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1432] Found device 1 with properties:
name: GeForce RTX 2080 major: 7 minor: 5 memoryClockRate(GHz): 1.83
pciBusID: 0000:65:00.0
totalMemory: 8.00GiB freeMemory: 6.55GiB
2019-09-03 16:13:53.406710: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1511] Adding visible gpu devices: 0, 1
2019-09-03 16:13:54.905149: I tensorflow/core/common_runtime/gpu/gpu_device.cc:982] Device interconnect StreamExecutor with strength 1 edge matrix:
2019-09-03 16:13:54.914554: I tensorflow/core/common_runtime/gpu/gpu_device.cc:988]      0 1
2019-09-03 16:13:54.918857: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1001] 0:   N N
2019-09-03 16:13:54.923367: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1001] 1:   N N
2019-09-03 16:13:54.926699: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1115] Created TensorFlow device (/job:localhost/replica:0/task:0/device:GPU:0 with 6294 MB memory) -> physical GPU (device: 0, name: GeForce RTX 2080, pci bus id: 0000:17:00.0, compute capability: 7.5)
2019-09-03 16:13:54.940144: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1115] Created TensorFlow device (/job:localhost/replica:0/task:0/device:GPU:1 with 6294 MB memory) -> physical GPU (device: 1, name: GeForce RTX 2080, pci bus id: 0000:65:00.0, compute capability: 7.5)
INFO:tensorflow:Restoring parameters from C:\Users\SakuLab2\Documents\TensorFlow\workspace\training_demo\pre-trained-model\model.ckpt
INFO:tensorflow:Restoring parameters from C:\Users\SakuLab2\Documents\TensorFlow\workspace\training_demo\pre-trained-model\model.ckpt
INFO:tensorflow:Running local_init_op.
INFO:tensorflow:Running local_init_op.
INFO:tensorflow:Done running local_init_op.
INFO:tensorflow:Done running local_init_op.
INFO:tensorflow:Starting Session.
INFO:tensorflow:Starting Session.
INFO:tensorflow:Saving checkpoint to path training/model.ckpt
INFO:tensorflow:Saving checkpoint to path training/model.ckpt
INFO:tensorflow:Starting Queues.
INFO:tensorflow:Starting Queues.
INFO:tensorflow:global_step/sec: 0
INFO:tensorflow:global_step/sec: 0
INFO:tensorflow:Recording summary at step 0.
INFO:tensorflow:Recording summary at step 0.

### Describe the problem
That's all.
I never see expected messages like
"INFO:tensorflow:global step 1: loss = 13.8886 (12.339 sec/step)
INFO:tensorflow:global step 2: loss = 9.8716 (9.339 sec/step)
INFO:tensorflow:global step 3: loss = 5.8716 (4.226 sec/step)
...".

I prepared 20 images, which was annotated 3-6 labels per image. All images are about 300x400 pix in jpg format. I additioned just one class. I divided these images into 16 train images and 4 test images.

What should I do? 
Please help me!

**Comment by SakuSakuLab** [comment]
I fixed it by re-installing tensorflow-gpu 1.14.0,  cuda10.0 and cuDNN 9.4.1.
Thank you!
